src/platform/packages/shared/kbn-connector-specs/README.md

```diff
@@ -147,9 +148,31 @@ metadata: {
 }
 ```
 
+### Auth Type
```

Comment:
1. How are we going to handle `none`?
    - Is it always appended to the existing `authTypes`? If so, what if the connector does not specify an `authType`?
    - Should we require it to always be explicitly defined?
2. Maybe I missed it(still reading the PR), but do we have a place that lists all available auth types?

Reply:
> Right now it's handled during connector registration, where if no auth types are specified, it defaults to z.object({}). We could be more explicit about it as well.

But should `none` also be part of `src/platform/packages/shared/kbn-connector-specs/src/all_auth_types.ts`?

`getAxiosInstanceWithAuth` has a flow for `authType: none`, but if that is not included when generating the schema s it won't ever happen, right?